react/typings/events.d.ts

```diff
@@ -140,21 +141,112 @@ export interface ProductImpressionData extends EventData {
   list: string
 }
 
+export interface CartData extends EventData {
+  event: 'cart'
+  eventName: 'vtex:cart'
+  orderForm: OrderForm
+}
+
+export interface AnalyticsEcommerceProduct {
+  id: string
+  name: string
+  category: string
+  brand: string
+  variant: string
+  price: number
+  quantity: number
+}
```

Comment:
I would move this to another file (related to Google), the intention of this file is to gather only the pixel events.